cal/cal/urls.py

```diff
@@ -16,6 +16,9 @@
         'cal.views.complete_with_token', name='complete_with_token'),
     url(r'^accounts/profile', 'cal.views.accounts_profile', name='accounts_profile'),
 
+    url(r'categories/(?P<pk>[0-9]+)/?', 'cal.views.category_detail', name='category_detail'),
+    url(r'tags/(?P<pk>[0-9]+)/?', 'cal.views.tag_detail', name='tag_detail'),
```

Comment:
@MaxFangX note: because we have URLs that start the same in both api and cal, cal/views.py is overriding api/views.py. I highly suggest we change the names of one or the other so that we can make post requests on categories and tags effectively.